cookbooks/travis_build_environment/templates/default/etc-systemd-conf-d-limits.conf.erb

```diff
@@ -0,0 +1,2 @@
+[Manager]
+DefaultLimitNOFILE=131072
```

Comment:
Do we want to increase maximum file open limit?

Reply:
2^16 would decrease the number, so I took 2^17. Happy for other suggestions.

On GCE images, I do not mind very high numbers. It might be wise to make this number configurable, and depend on whether this runs on docker with shared resources with the host or not.